frontend/style/progress.css

```diff
@@ -18,10 +18,27 @@
   width: 20vw;
 }
 
+.image {
+  width: 12vw;
+  padding: 1vw;
+  border: 0.25vw solid gray;
+}
+
 #progress-container {
   display: none;
 }
 
 #download-btn {
   display: none;
 }
+
+footer {
+  position: absolute;
```

Comment:
@bernep can you use fixed here instead of absolute

Reply:
Yeah.